ansible/roles/svc_minio/tasks/setup_minio_users.yml

```diff
@@ -0,0 +1,75 @@
+---
+# code: language=ansible
+# Path: ansible/roles/svc_minio/tasks/setup_minio_users.yml
+# This file contains the tasks to create deerhide-operator users for Minio
+
+- name: Check list of users in minio
+  ansible.builtin.shell:
+    cmd: |
+      set -eo pipefail
+      mc admin user list melanie --json | jq -s .
+    executable: /bin/bash
+  changed_when: false
+  register: minio_users_list
+
+- name: Set fact and generate password
+  ansible.builtin.set_fact:
+    svc_minio_users:
+      - { name: "mirage", gpg_userid: "{{ melanie_gpg_userid_mirage }}", userid: "{{ slack_mirage }}",
+          present: "{{ minio_users_list.stdout | from_json | community.general.json_query(check_mirage) | bool }}",
+          password: "{{ lookup('ansible.builtin.password', '/dev/null', chars=['ascii_letters', 'digits'], length=15) }}", } # noqa: jinja[invalid]
+      - { name: "nakou", gpg_userid: "{{ melanie_gpg_userid_nakou }}", userid: "{{ slack_nakou }}",
+          present: "{{ minio_users_list.stdout | from_json | community.general.json_query(check_nakou) | bool }}",
+          password: "{{ lookup('ansible.builtin.password', '/dev/null', chars=['ascii_letters', 'digits'], length=15) }}", } # noqa: jinja[invalid]
+      - { name: "tom", gpg_userid: "{{ melanie_gpg_userid_tom }}", userid: "{{ slack_tom }}",
+          present: "{{ minio_users_list.stdout | from_json | community.general.json_query(check_tom) | bool }}",
+          password: "{{ lookup('ansible.builtin.password', '/dev/null', chars=['ascii_letters', 'digits'], length=15) }}", } # noqa: jinja[invalid]
+  vars:
+    check_mirage: "[?status==`success`] | contains([].accessKey, `mirage`)"
+    check_nakou: "[?status==`success`] | contains([].accessKey, `nakou`)"
+    check_tom: "[?status==`success`] | contains([].accessKey, `tom`)"
```

Comment:
@miragecentury - can we simplify this block? Since the only variable is the username, we could probably come up with a more future-proof solution.

Reply:
For this block yes but, i use all after

Reply:
To do later